src/main/kotlin/net/axay/kotlinkitapi/implementation/Automatic.kt

```diff
@@ -0,0 +1,31 @@
+package net.axay.kotlinkitapi.implementation
+
+import net.axay.kotlinkitapi.api.Kit
+import net.axay.kotlinkitapi.api.KitProperties
+import net.axay.kotlinkitapi.utils.isSoup
+import net.axay.kspigot.extensions.broadcast
+import org.bukkit.attribute.Attribute
+import org.bukkit.entity.Player
+import org.bukkit.event.entity.EntityDamageEvent
+import kotlin.math.min
+
+class AutomaticProperties : KitProperties() {
+    val soupHealAmount by double(5.0)
```

Comment:
shouldn't automatic heal the same as normal soups do, in that case maybe change this to
```kt
val soupHealAmount by any<Double?>(null)
```
where null means using the default soup healing amount

Reply:
AFAIK it doesn't; old kitapi also healed the player for 5HP

Reply:
yeah but I would do it this way
also, it would still be configurable in the config or via a command